src/heitech.zer0mqXt.core/IZer0MqBuilder.cs

```diff
@@ -11,6 +11,7 @@ public interface IZer0MqBuilder
     {
         //todo create interface to replace services as soon as an injector is decided
         IZer0MqBuilder SetLogger(ILogger adapter);
+        IZer0MqBuilder SilentLogger();
```

Comment:
to me, the method name reads like a type name - perhaps `SilenceLogger` would be a better fit?

Reply:
Good point!